### ISSUE TITLE
Show version number in settings and allow copying

### ISSUE THREAD

**Issue by thegabriestein** [ffMathy/Shapeshifter#342]
When manually submitting bugs or ideas, would be really nice to have the version number listed in the menu in larger font with a clickable "Copy" button so you could reference an exact build number in an issue like this:

![](https://i.imgur.com/jQQOmtE.png)

**Comment by ffMathy** [comment]
The version number is already displayed when hovering the tray icon, but not in a copyable format no.

What I would much rather do though and turn this issue into is build on top of #226 and have a UI that can submit an issue. This would then also embed the logs automatically and report the version etc. After reporting, it could open the issue in a browser.

How does that sound?

**Comment by thegabriestein** [comment]
Yeah that works. As long as it's easy to reference a build number when manually creating issues. Because a number of times the application is crashing or having issues and I can't get to the settings to get the version number, so I have to open the exe properties or other hassle to get it. Just need to make that easier.

**Comment by ffMathy** [comment]
I think for the cases where automatic submission wouldn't work, we would survive with just reporting the number manually.

Hopefully we will get to a point where that happens very rarely or ever, and if it does not, we have a whole bigger problem.
